### ISSUE TITLE
No API for getting contig/chromosome sequence?

### ISSUE THREAD

**Issue by erasche** [GMOD/Apollo#705]
As far as I can tell, only the annotated feature sequences are accessible via the API. Would it be possible to expose a route for downloading the complete chromosome sequence as fasta?

`/organism/getSequencesForOrganism` was the closest I could find, it gives me metadata about a sequence but doesn't actually offer an API for fetching the actual sequence object's DNA.

```
{
    "username": "hxr@tamu.edu", 
    "organism": [
        "test"
    ], 
    "sequences": [
        {
            "seqChunkSize": 20000, 
            "end": 172788, 
            "name": "Merlin", 
            "featureLocations": [], 
            "id": 66, 
            "start": 0, 
            "length": 172788, 
            "sequenceChunks": [], 
            "organism": {
                "class": "org.bbop.apollo.Organism", 
                "id": 64
            }, 
            "class": "org.bbop.apollo.Sequence"
        }
    ]
}
```

**Comment by nathandunn** [comment]
Did you look in IO Services? 

http://icebox.lbl.gov/Apollo-staging/WebServices/ <http://icebox.lbl.gov/Apollo-staging/WebServices/>


There is an example (get_gff3.groovy):



Nathan

**Comment by erasche** [comment]
Ah, that data is probably available in #704. Nevermind then. Thanks :)
